src/reduxStore/persistStorage.ts

```diff
@@ -1,15 +1,13 @@
 import reduxPersistLocalStorage from 'redux-persist/lib/storage';
 import reduxPersistSessionStorage from 'redux-persist/lib/storage/session';
-import AsyncStorage from '@react-native-async-storage/async-storage';
-import { isMobileEnvironment } from '../utils';
 
 const persistLocalStorage = reduxPersistLocalStorage;
 const persistSessionStorage = reduxPersistSessionStorage;
 
 export const getPersistLocalStorage = () => {
```

Comment:
I think we can remove this file and just import these storages into the persistReducer file